spec/features/user/auth_user_cannot_be_admin_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.